flow/raster_cache.cc

```diff
@@ -118,8 +123,10 @@ RasterCacheResult RasterizePicture(SkPicture* picture,
 
   return {
       surface->makeImageSnapshot(),  // image
-      physical_rect,                 // source rect
-      logical_rect                   // destination rect
+      // compensate canvas->translate(-logical_rect.left(), -logical_rect.top())
```

Comment:
Did you forget to run `clang-format` over your patch?

Reply:
Yes. Running it now.